code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -1117,9 +1118,18 @@
 /mob/living/simple_animal/proc/PunchTarget()
 	if(!Adjacent(target_mob))
 		return
+	sleep(rand(8) + 8)
 	if(isliving(target_mob))
 		var/mob/living/L = target_mob
-		L.attack_generic(src,rand(melee_damage_lower,melee_damage_upper),attacktext)
+
+		if(prob(melee_miss_chance))
+			src.attack_log += text("\[[time_stamp()]\] <font color='red'>attacked [L.name] ([L.ckey])</font>")
+			L.attack_log += text("\[[time_stamp()]\] <font color='orange'>was attacked by [src.name] ([src.ckey])</font>")

```

Comment:
Regular unarmed attacks for players, iirc, log misses, so included here for sake of completeness.